Library/Formula/logstash.rb

```diff
@@ -20,6 +22,6 @@ def caveats; <<-EOS.undent
 
   test do
     system "#{bin}/logstash", "--version"
-    system "/bin/echo 'hello world' | #{bin}/logstash agent -e 'input { stdin { type => stdin } } output { stdout { codec => rubydebug } }'"
+    system "/bin/echo", "'hello world' | #{bin}/logstash agent -e 'input { stdin { type => stdin } } output { stdout { codec => rubydebug } }'"
```

Comment:
Can we get some movement on this? Seems to be the only thing halting this being merged right now. Thanks!